src/fileWriters/index.js

```diff
@@ -1,12 +1,50 @@
-import path from 'path';
+import extractZip from 'extract-zip';
+import chalk from 'chalk';
+import axios from 'axios';
+import fse from 'fs-extra';
+import rimraf from 'rimraf';
 
 import writeTemplateDirectoryFiles from './writeTemplateDirectoryFiles';
+import createTemplateFiles from './createTemplateFiles';
+import {
+  SOURCE_URL,
+  SOURCE_FILENAME,
+  TEMP_DIR,
+  TEMPLATES_DIR,
+} from '../constants';
 
-const writeBaseTemplates = async ({ destination, templateValues }) =>
+export const writeBaseTemplates = async ({ destination, templateValues }) =>
   writeTemplateDirectoryFiles({
     destination,
     templateValues,
-    location: path.join(__dirname, '../templates'),
+    location: TEMPLATES_DIR,
   });
 
-export default writeBaseTemplates;
+const extractSource = async (source) => {
+  extractZip(source, { dir: TEMP_DIR }, async (error) => {
+    if (error) {
+      console.log(chalk.bold.redBright(error));
+      return;
+    }
+    await createTemplateFiles();
+  });
+};
+
+export const getCookieCutterSource = async () => {
+  console.log(chalk.bold.blueBright(`🐶  Fetching source from ${SOURCE_URL}...`));
+
+  const response = await axios({
+    url: SOURCE_URL,
+    method: 'get',
+    responseType: 'arraybuffer',
+  });
+
+  if (fse.existsSync(TEMP_DIR)) {
+    rimraf.sync(TEMP_DIR);
```

Comment:
`fse` has [a `remove` method](https://github.com/jprichardson/node-fs-extra/blob/master/docs/remove.md#removepath-callback) - I'm not sure you need `rimraf`.

Reply:
Gotcha, I'm not too familiar with the methods available in `fse`. I'll swap this over.